basic_ui/MiniMediaPlayer.qml

```diff
@@ -109,29 +109,65 @@ Item {
 
         onButtonPressed: {
             switch (button) {
-            case ButtonHandler.DPAD_MIDDLE:
-                if (entities.mediaplayersPlaying[mediaPlayers.currentIndex].state === MediaPlayer.PLAYING ) {
-                    entities.mediaplayersPlaying[mediaPlayers.currentIndex].pause();
-                } else {
-                    entities.mediaplayersPlaying[mediaPlayers.currentIndex].play();
```

Comment:
Where are these removed things are going to be handled then?

Reply:
In each integration they should be handled individual. Due to the fact that a play/pause/next/previous button is already on the UI, it makes no sense to handle them twice?

Reply:
Exactly, so all the integrations should also have a PR that adds this functionality. Making this change without fixing the integrations will break all of them.

Reply:
No integration will Be break down due to fact that the middle button will not send an play/pause command anymore but the touch button still. So it's just remove the hardware button behaviour.

Reply:
Right, but this change will remove a functionality that was existed before :)

Reply:
Also not because of you have different Mediaplayers entitys you can't use it due to fact the remote doesn't need know which Mediaplayer is mentioned :(

Reply:
It knows: `entities.mediaplayersPlaying[mediaPlayers.currentIndex]`

Reply:
If I want to pause Kodi and not Spotify but Spotify started later Spotify will be paused and not Kodi.

Reply:
reverted back